src/sentry/api/endpoints/sentry_app_installation_external_issues.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases import SentryAppInstallationBaseEndpoint
+from sentry.api.serializers import serialize
+from sentry.mediators.external_issues import IssueLinkCreator
+from sentry.models import Group
+
+
+class SentryAppInstallationExternalIssuesEndpoint(SentryAppInstallationBaseEndpoint):
+    def post(self, request, installation):
+        if not features.has('organizations:sentry-apps',
+                            installation.organization,
+                            actor=request.user):
+            return Response(status=404)
+
+        group_id = request.DATA.get('groupId')
+        if not group_id:
+            Response({'detail': 'groupId is required'})
+
+        try:
+            group = Group.objects.get(id=group_id)
```

Comment:
We should be doing something like:

```python
Group.objects.get(
    id=group_id,
    project__in=Project.objects.filter(
        organization_id=installation.organization_id,
    )
)
```

to ensure someone doesn't attempt to call this endpoint with a rando `groupId`, potentially affecting other people's data.